app/pages/adduser/confirmation/ui.test.js

```diff
@@ -5,7 +5,7 @@ import { extractInnerText } from '../../../test-utils/helper';
 import { organisationsApiLocalhost } from '../../../test-utils/config';
 import organisationDetails from '../../../test-utils/fixtures/organisationDetails.json';
 
-const path = 'http://localhost:1234/organisations/org1/adduser/confirmation?id=user1';
+const path = 'http://localhost:1234/admin/organisations/org1/adduser/confirmation?id=user1';
```

Comment:
Do we need to put baseUrl in here?

Reply:
I've decided to hard code it in the ui.test to make it explicit this is what the page we are testing is